parm/config/gfs/config.base.emc.dyn

```diff
@@ -214,7 +214,7 @@ case "${APP}" in
     export DO_COUPLED="YES"
     export DO_OCN="YES"
     export DO_ICE="YES"
-    export CCPP_SUITE="FV3_GFS_v17_coupled_p8"  # TODO: Does this include FV3_GFS_v17_p8?  Can this be used instead of FV3_GFS_v17_p8 on L149
+    export CCPP_SUITE="FV3_GFS_v17_coupled_p8_ugwpv1"  # TODO: Does this include FV3_GFS_v17_p8?  Can this be used instead of FV3_GFS_v17_p8 on L149
```

Comment:
Can someone answer the question on #TODO here?  Also, note the line has changed from L149 to 151.

Reply:
I remember this was an open question.  I wanted to say that Moorthi would run the "cpld" CCPP suite with the uncoupled atm-only, but the details are fuzzy. Based on memory, the coupled CCPP suite files have one extra call incase you have an ice model, but otherwise are the same.  I think that what is currently here is consistent with the way things are run in regression tests.

Reply:
I'd recommend retesting just incase my memory is bad or in case things have since changed.

Reply:
Great. Can someone on the modeling team verify that it runs and produces acceptable results?
It does not have to be done for this PR to be merged.  Do however, update the comment and if possible, add that testing will be done to resolve the `TODO`.

Reply:
I went to look at the ufs-weather-model regression tests and how the workflow is now is consistent with how I saw the regression tests being run, with different CCPP suite files for coupled versus uncoupled.  While it's possible I missed something, if I did not,  is there a reason to move away from how the regression tests are run with the two separate CCPP suite files for the coupled or uncoupled cases?

Reply:
I don't think the UFSWM RT and global-workflow are consistent in general.  Yes, they are consistent in that they both use the different CCPP suite file for coupled and uncoupled.  However, the question I asked is, is that necessary?  Should they be the same or should they be different? 
From the perspective of unification, an uncoupled configuration is a coupled model with all the components OFF.
I am not expecting a response for that question in this PR, as the answer will require further investigation.